Sources/BraintreeUIComponents/BTPaymentButtonView.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+/// Shared payment button view that works with any PaymentButtonColorProtocol
+///  Not to be called directly. Use either BTPayPalButtons or BTVenmoButton instead.
+struct PaymentButtonView<Color: PaymentButtonColorProtocol>: View {
```

Comment:
I'm not very familiar with how visibility modifiers work in swift, is there a way we can make this class internal to the SDK so that merchants aren't able to reference this type?